myscript.js

```diff
@@ -0,0 +1,21 @@
+
+var dataArray = [1, 8, 7, [ 4, 65, 1, [ 2, 7 ], 45, [ 6, 4], 6 ], 4, 8]
+
+var jenyaFlatternArray = function (Array) {
```

Comment:
there is a convention to have name of variables sytarting with lower case, like this `myVariable`. Then classes should be named with capital letter, like this:
```
var Person = function(name) {};
var me = new Person('Vova')
```